help/develop/domains/chem/cheminformatics.md

```diff
@@ -109,38 +86,34 @@ columns ([code sample](https://public.datagrok.ai/js/samples/domains/chem/simila
 Produces a Datagrok [`DataFrame`](https://datagrok.ai/js-api/classes/dg.DataFrame) with a
 single [`Column`](https://datagrok.ai/js-api/classes/dg.Column), where the i-th element contains a similarity score for
 the i-th element of the
-input `Column` ([code sample](https://public.datagrok.ai/js/samples/domains/chem/similarity-scoring-scores))
-.
+input `Column` ([code sample](https://public.datagrok.ai/js/samples/domains/chem/similarity-scoring-scores)).
 
-Similarly to the substructure-library based method of `searchSubstructure`, these functions maintain a cache of
+Similarly to the `searchSubstructure` function, these functions maintain a cache of
 pre-computed fingerprints. Once the function `findSimilar` or `getSimilarities` meets the `column` previously unmet, it
-builds the cache of computed fingerprints for the molecules of this `column`, and later uses it to compute similarity
+builds the cache of computed fingerprints for the molecules in `column`. Later it is used to compute similarity
 scores as long as the function is invoked for the same `column`.
 
 To only build (aka prime) this cache without performing an actual search, e.g. in case of pre-computing the fingerprints
-in the UI, the function should be called passing `molecule` either as `null` or as an empty string `''`.
+in the UI, call the function passing `molecule` as an empty string `''`.
 
-Sometimes it happens that the molecule strings aren't supported by RDKit-JS. These inputs are skipped when indexing, but
-an error log entry is added for each such entry to the browser's console together with the actual molecule string. For
-such unsupported molecule, the corresponding fingerprint of all zeroes is produced. Thus, the output Column (or
-DataFrame) is *always* of the input `column`'s length (with the number of rows equal to `column`'s length, respectively)
+In case molecule strings are not supported by RDKit JS they are skipped when indexing. For
```

Comment:
`If molecules are not supported by RDKit JS, we skip them durring indexing`

Reply:
Done